### DISCUSSION TITLE
Github Copiloto integrado no visual studio 17.10 não está funcionando como esperado

### DISCUSSION THREAD

**Discussion by martaoliveirasi** [community/community#126711] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Github  integrado no visual studio 17.10 é habiltiado mas não funciona como esperado, não responde, na versão anterior do visual studio o chat copilot era instalado e acoplado e funcionava perfeitamente.

**Comment by ebndev** [comment]
Hi @martaoliveirasi, thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting to ensure discussions are adhering to our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md). 

We’ll be locking this post for now, but please feel encouraged to create a new discussion in English if you still need support.
